examples/setup_assistant.py

```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/python
+"""Nibbler."""
+
+from nibbler import *
+
+import objc
+from AppKit import NSTimer
+from OpenDirectory import *
+
+import argparse
+import logging
+import os
+
+# black magic
+MapKit = objc.loadBundle(
+  'MapKit',
+  globals(), bundle_path='/System/Library/Frameworks/MapKit.framework'
+)
```

Comment:
It's in the sam_example.py script, but doesn't seem to be used there. Wasn't sure if it actually did some 'black magic' as the comment in sam_example.py implies.